<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15, 125, 209);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 26pt &quot;MS Shell Dlg 2&quot;;color: rgb(255, 255, 255);

</string>
    </property>
    <property name="text">
     <string>CLIENTES</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>DNI:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDni">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nombres:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombres">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Apellido Paterno:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellidoPaterno">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Apellido Materno:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellidoMaterno">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Dirección:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDireccion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>611</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTelefono">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>311</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Teléfono:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblClientes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>931</width>
      <height>291</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <column>
     <property name="text">
      <string>DNI</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nombres</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string notr="true">Apellido Paterno</string>
     </property>
     <property name="whatsThis">
      <string notr="true"/>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <stylestrategy>PreferAntialias</stylestrategy>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Apellido Materno </string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Dirección</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Teléfono</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <stylestrategy>NoAntialias</stylestrategy>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnRegistrar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(235, 38, 12);</string>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/registrar.png</normaloff>../IMG/registrar.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModificar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>540</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);background-color: rgb(235, 38, 12);</string>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/modificar.png</normaloff>../IMG/modificar.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConsultar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>540</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);background-color: rgb(235, 38, 12);</string>
    </property>
    <property name="text">
     <string>Consultar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/consultar.png</normaloff>../IMG/consultar.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>540</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);background-color: rgb(235, 38, 12);</string>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/eliminar.png</normaloff>../IMG/eliminar.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnListar">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>540</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);background-color: rgb(235, 38, 12);</string>
    </property>
    <property name="text">
     <string>Listar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/listar.png</normaloff>../IMG/listar.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
